hdl/modules/fofb_sys_id/cheby/wb_fofb_sys_id_regs.cheby

```diff
@@ -28,3 +28,153 @@ memory-map:
+
+                write 0x0000: new PRBS iteration at each FOFB cycle
+                write 0x0001: new PRBS iteration at each 2 FOFB cycles
+                ...
+                write 0x03FF: new PRBS iteration at each 1024 FOFB cycles
+          - reg:
+              name: lfsr_length
+              width: 8
+              access: rw
+              address: next
+              description: |
+                Length of internal LFSR (max: 0x1E)
+                Together with step_duration, defines the duration of PRBS,
+                which is given by [2^(lfsr_length + 2) - 1]*(step_duration + 1)
+
+                write 0x00: set LFSR length to 2
+                write 0x01: set LFSR length to 3
+                ...
+                write 0x1E: set LFSR length to 32
+          - block:
+              name: bpm_pos_distort
+              description: Interface to prbs_bpm_pos_distort regs
+              address: next
+              children:
+                - reg:
+                    name: ctl
+                    width: 32
+                    access: rw
+                    address: next
+                    description: |
+                      PRBS-based distortion for BPM positions control register
+                    children:
+                      - field:
+                          name: en_distort
+                          range: 0
+                          description: |
+                            Enables/disables PRBS-based distortion on BPM
+                            positions
+
+                            write 0: distortion disabled
+                            write 1: distortion enabled
+                - memory:
+                    name: distort_ram
+                    memsize: 2k
+                    address: next
+                    description: Distortion levels RAM
+                    children:
+                      - reg:
+                          name: levels
+                          width: 32
+                          access: rw
+                          description: |
+                            Two S16FP0-bit distortion levels, one for each PRBS
+                            value.
+
+                            15 - 0: distortion level for PRBS value 0
+                            31 - 16: distortion level for PRBS value 1
+                          children:
+                            - field:
+                                name: level_0
+                                range: 15-0
+                                description: |
+                                  Distortion level for PRBS value 0 in
+                                  nanometers
+                            - field:
+                                name: level_1
+                                range: 31-16
+                                description: |
+                                  Distortion level for PRBS value 1 in
+                                  nanometers
```

Comment:
Move the memory block to the end of the cheby file to minimize the amount of padding required.